content/800-data-platform/06-query-console.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: 'Query Console'
+metaTitle: 'Query Console'
+metaDescription: 'Query Console'
+toc: true
+tocDepth: 3
+---
+
+<TopBlock>
+
+Use the Query Console to run the 3 pre-defined `findMany()` test queries or add and test queries of your own using the [Prisma Client API for model queries](/reference/api-reference/prisma-client-reference#model-queries).
+
+</TopBlock>
+
+## UI components
+
+The Query Console is comprised of 2 major UI components - **Query Editor** and **Query Results Console**.
+
+<img src="https://res.cloudinary.com/prismaio/image/upload/v1657148399/docs/pdp/pdp-query-console-components_w98dp5.png" alt="Query Console - UI Components"/>
+
+| UI component          | Description                                                                         |
+| --------------------- | ----------------------------------------------------------------------------------- |
+| Query Editor          | • Add, edit, delete queries using the Prisma Client API<br />• Run a selected query |
+| Query Results Console | View the results from the selected query                                            |
+
+## Default model queries
+
+- `findMany()` for the first 100 records in the `User` model
+- `findMany()` for the first 100 records in the `User` model that match a criteria
+- `findMany()` for the first 100 records in the `User` model but only return data from 2 fields
```

Comment:
Is this always the `User` query or not just the first one or something like that? DevProd team can probably tell you.

Reply:
Added a note that this is from a project that specifically has the `User` model.